src-lib/image_opencv.cpp

```diff
@@ -82,12 +82,30 @@ extern "C" void resize_window_cv(char const* window_name, int width, int height)
 
 extern "C" {
 
-mat_cv * load_image_mat_cv(const char * const filename, int flag)
+mat_cv * load_image_mat_cv(const char * const filename, int channels)
 {
 	cv::Mat * mat = nullptr;
 
 	try
 	{
+		int flag = cv::IMREAD_UNCHANGED;
+		if (channels == 0)
```

Comment:
Does this make sense?  Why would channel be set to zero?  Shouldn't this be a fatal error?

Reply:
It does not - it was part of a copy-and-paste from the original helper method.  I'll remove it.